src/components/NotesComponent/NotesComponent.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import ListComponent from '../ListComponent/ListComponent';
+import SearchComponent from '../SearchComponent/SearchComponent';
+
+const notes = [
+  {
+    title: 'Note one',
+    body: 'This is the note one.',
+    id: '1'
+  },
+  {
+    title: 'Note two',
+    body: 'This is the note two.',
+    id: '2'
+  },
+  {
+    title: 'Burek',
+    body: 'This is the note three.',
+    id: '3'
+  },
+];
+
+class NotesComponent extends Component {
+  state = {
```

Comment:
This is not the proper way to set initial state, please do it this way:
constructor(props) {
    super(props);
    this.state = { 
          //initialize state here
    };
  }

Reply:
Just a small correction, it IS a 'proper' way, but not the best way to do it, so define state in constructor.